lib/std/zig/parser_test.zig

```diff
@@ -1801,19 +1801,19 @@ test "zig fmt: array literal with hint" {
     );
 }
 
-test "zig fmt: array literal veritical column alignment" {
+test "zig fmt: array literal vertical column alignment" {
     try testTransform(
         \\const a = []u8{
         \\    1000, 200,
         \\    30, 4,
-        \\    50000, 60
+        \\    50000, 60,
         \\};
         \\const a = []u8{0,   1, 2, 3, 40,
         \\    4,5,600,7,
         \\           80,
-        \\    9, 10, 11, 0, 13, 14, 15};
+        \\    9, 10, 11, 0, 13, 14, 15,};
         \\const a = [12]u8{
-        \\    31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
+        \\    31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31, };
```

Comment:
could we not change this test case? it was added to catch this bug: https://github.com/ziglang/zig/issues/8423

Reply:
I had to add the tailing comma for the same code path to be exercised, but I've copied the same case minus the trailing comma anyway.